src/error.rs

```diff
@@ -64,8 +64,6 @@ pub enum Error {
   CustomProtocolTaskInvalid,
   #[error("Failed to register URL scheme: {0}, could be due to invalid URL scheme or the scheme is already registered.")]
   UrlSchemeRegisterError(String),
-  #[error("Duplicate custom protocol registered on Linux: {0}")]
-  DuplicateCustomProtocol(String),
```

Comment:
This error needs to be brought back, I'd rather emit an error than silently ignore one of the calls to `with_custom_protocol`

Reply:
I'm fine with that, removed this mainly because the error says 'Duplicate custom protocol registered on Linux' which I would assume to be the same as the one below

About if we should allow duplicates in the same builder though, I don't mind it either way

Reply:
> I'm fine with that, removed this mainly because the error says 'Duplicate custom protocol registered on Linux' which I assumed to be the same as the one below, since there're technically no restrictions to register it multiple times in a builder

yeah that wording was just incorrect